SPECS/glusterfs/include-eventtypes-always.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/libglusterfs/src/glusterfs/events.h b/libglusterfs/src/glusterfs/events.h
+index 68b7996519..8874698bb2 100644
+--- a/libglusterfs/src/glusterfs/events.h
++++ b/libglusterfs/src/glusterfs/events.h
+@@ -11,8 +11,8 @@
+ #ifndef __EVENTS_H__
+ #define __EVENTS_H__
+ 
+-#ifdef USE_EVENTS
+ #include "eventtypes.h"
++#ifdef USE_EVENTS
```

Comment:
i dont think it can be handled that way.  the code has a #if and a #else, both of which require eventtypes.h.

Reply:
glusterfs in mariner 2.0 built with disable-events, when doing that for 3.0, the header file is not included (despite being needed) without this patch.

Reply:
Well I'll be, indeed it looks like a bug in the upstream ([this change](https://github.com/gluster/glusterfs/commit/3b00855b0f69e4cc36160d56b36f6b011bdd1177#diff-aead3630bb5df6aed67b31bea331412c2c900f295e76a8f369e4a84096783880) might be the culprit). I was not expecting that. In that case I'd only ask to make the comment above this patch in the spec more robust, explaining in more detail what was wrong (I understand it's actually impossible to compile correctly with `--disable-events`), maybe with the above commit link if you also believe this may be the source of the issue. Ideally, please also open an issue in upstream's GitHub to link it here.